src/lib.rs

```diff
@@ -1045,6 +1048,19 @@ impl Client {
         self.current_url_().map(|(_, u)| u)
     }
 
+    /// Get a PNG screenshot of the current page.
+    pub fn screenshot(&self) -> impl Future<Item = Vec<u8>, Error = error::CmdError> + 'static {
+        self.dup()
+            .issue_wd_cmd(WebDriverCommand::TakeScreenshot)
+            .and_then(|(_, src)| {
+                if let Some(src) = src.as_string() {
+                    return decode(src).map_err(error::CmdError::from);
```

Comment:
I think I'd prefer `base64::decode` here rather than the top-of-file `use base64::decode`.